api/swagger.yaml

```diff
@@ -2190,6 +2190,26 @@ definitions:
         type: "string"
         x-nullable: false
 
+  PluginPrivilegeItem:
+    description: |
+      Describes a permission the user has to accept upon installing
+      the plugin.
+    type: "object"
+    x-nullable: false
+    required: [Name, Value]
```

Comment:
I hope adding the `x-nullable: false` and the `required` contrains don't break anything - or do they?

Reply:
Does it make sense to leave it for a separate PR? That could (a bit) simplify the review at least

Reply:
I've removed the change and will create another PR.